src/Text/Parser/Combinators.agda

```diff
@@ -1,6 +1,8 @@
 {-# OPTIONS --without-K --safe #-}
 
-module Text.Parser.Combinators where
+open import Text.Parser.Types.Core using (Parameters)
+
+module Text.Parser.Combinators {l} {P : Parameters l} where
```

Comment:
Why have you decided to make the module parameters implicit?

Reply:
I had started with an explicit parameter but then if you open
`Text.Parser.Combinators` unapplied (which ought to be backwards
compatible) then you get infix operators that take 3 visible
arguments instead of 2!